### PR TITLE
fix(@desktop/communities): Fix community tokens model

### LINKED ISSUE #9781: Fix exposing fields in community tokenModel
Some fields are not exposed: symbol, chainId.

### PR DESCRIPTION
**Pull request by endulab**
Fix #9781

### What does the PR do

Expose missing fields

### Screenshot of functionality (including design for comparison)

image

### CODE REVIEW

File: src/app_service/service/community_tokens/service.nim [hunk 1]
```diff
@@ -77,9 +77,9 @@ QtObject:
       let contractGasUnits = "3702411"
 
       let txData = ens_utils.buildTransaction(parseAddress(addressFrom), 0.u256, contractGasUnits,
-      if suggestedFees.eip1559Enabled: "" else: $suggestedFees.gasPrice, suggestedFees.eip1559Enabled,
-      if suggestedFees.eip1559Enabled: $suggestedFees.maxPriorityFeePerGas else: "",
-      if suggestedFees.eip1559Enabled: $suggestedFees.maxFeePerGasM else: "")
+        if suggestedFees.eip1559Enabled: "" else: $suggestedFees.gasPrice, suggestedFees.eip1559Enabled,
+        if suggestedFees.eip1559Enabled: $suggestedFees.maxPriorityFeePerGas else: "",
+        if suggestedFees.eip1559Enabled: $suggestedFees.maxFeePerGasM else: "")
```
Comment (0x-r4bbit) [inline src/app_service/service/community_tokens/service.nim:82]: The indentation looks a bit off here

**Comment by 0x-r4bbit** [comment]
Looks like this needs a rebase

File: src/app/modules/main/communities/minting/models/token_model.nim [hunk 2]
```diff
@@ -104,7 +108,7 @@ QtObject:
         result = newQVariant(item.transferable)
       of ModelRole.RemoteSelfDestruct:
         result = newQVariant(item.remoteSelfDestruct)
-      of ModelRole.NetworkId:
+      of ModelRole.ChainId:
```
Comment (noeliaSD) [inline src/app/modules/main/communities/minting/models/token_model.nim:111]: And `image`?

**Review by noeliaSD — COMMENTED** [review]
Minor comment added!

File: src/app_service/service/community_tokens/service.nim (hunk 1 above)
Comment (endulab) [reply src/app_service/service/community_tokens/service.nim:82]: Jo asked about it in the prev pr to indicate that these lines belongs to txData.

File: src/app/modules/main/communities/minting/models/token_model.nim (hunk 2 above)
Comment (endulab) [reply src/app/modules/main/communities/minting/models/token_model.nim:111]: It is already in the bottom

Comment (noeliaSD) [reply src/app/modules/main/communities/minting/models/token_model.nim:111]: True! didn't see it before!

**Review by noeliaSD — APPROVED** [review]

File: src/app_service/service/community_tokens/service.nim (hunk 1 above)
Comment (0x-r4bbit) [reply src/app_service/service/community_tokens/service.nim:82]: ohh I see ti now

**Review by 0x-r4bbit — APPROVED** [review]

**Pull request merged**